statement_test.go

```diff
@@ -171,6 +255,20 @@ func TestPrepareQueryPositional(t *testing.T) {
 		require.ErrorContains(t, innerErr, paramIndexErrMsg)
 		require.Equal(t, TYPE_INVALID, paramType)
 
+		// Test column methods for UPDATE statement (should have no columns)
+		columnCount, innerErr := stmt.ColumnCount()
+		require.NoError(t, innerErr)
+		require.Equal(t, 1, columnCount)
+
+		// Test out of bounds access - should return empty/invalid for UPDATE with no columns
+		colName, innerErr := stmt.ColumnName(0)
+		require.NoError(t, innerErr)
+		require.Equal(t, "Count", colName)
+
+		colType, innerErr := stmt.ColumnType(0)
+		require.NoError(t, innerErr)
+		require.Equal(t, TYPE_BIGINT, colType)
```

Comment:
Do the comments match the tests here? We compare to Count, so it is not empty/invalid? Same for `TYPE_BIGINT`?